.github/workflows/smoke_tests.yml

```
@@ -36,9 +37,64 @@ jobs:
       - name: Ize build
```

It's all about ize, so let's not abuse the name.
```suggestion
      - name: Build
```